tests/test_evaluation.py

```
@@ -264,16 +281,13 @@ def create_labels_no_match_frame_pairs(minimal_instance):
+def test_evaluator_no_match_frame_pairs(caplog, minimal_instance):
     # with no match frame pairs
-
     user_labels, pred_labels = create_labels_no_match_frame_pairs(minimal_instance)
-    # there should not be any match frame pairs as the video names are different.
-    # test if exception is raised in case of empty frame pairs
-    with pytest.raises(
-        Exception, match="Empty Frame Pairs. No match found for the video frames"
-    ):
-        eval = Evaluator(user_labels, pred_labels)
+    with caplog.at_level("ERROR"):  # Set the log level to capture ERROR messages
+        with pytest.raises(Exception):
```

Ruff (0.8.2)

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)